tests/ctst/common/common.ts

```diff
@@ -33,11 +33,11 @@ Given('an existing bucket {string} {string} versioning', async function (bucketN
     }
 });
 
-Given('an object that {string} {string} version-Id',
-    async function (objectExists, withVersioning) {
+Given('an object {string} that {string} {string} version-Id',
+    async function (objectName, objectExists, withVersioning) {
     this.resetCommand();
     if (objectExists === 'exists') {
-        this.saved.objectName = 'x'.repeat(10);
+        this.saved.objectName = objectName || 'x'.repeat(10);
```

Comment:
IMHO the default value here should be something random, the repeat could be used as an object body though. It will allow us not to have any conflict in the scenario where the object name is not specified, what do you think?

Reply:
Reopening as I do not see any change for this comment (or reason why not to change the logic)